src/common/Tile/index.js

```diff
@@ -11,22 +11,30 @@ import {
   VoteAverage,
   VoteCount,
 } from "./style";
-import starIcon from "./images/Vector.svg";
 
-const Tile = ({ title, year, type, imagePath, voteAverage, voteCount }) => {
+const Tile = ({
+  type,
+  title,
+  year,
+  group,
+  imagePath,
+  starIcon,
+  voteAverage,
+  voteCount,
+}) => {
   return (
-    <Container>
-      <Poster src={imagePath} alt={""} />
-      <Content>
+    <Container people={group}>
+      <Poster people={group} src={imagePath} alt={""} />
+      <Content people={group}>
         <DetailsContainer>
-          <Title>{title}</Title>
+          <Title people={group}>{title}</Title>
           <Year>{year}</Year>
           <Tags>{type}</Tags>
         </DetailsContainer>
         <VoteContainer>
-          <img src={starIcon} alt={"star"}></img>
+          <img src={starIcon} alt={""}></img>
           <VoteAverage>{voteAverage}</VoteAverage>
-          <VoteCount>{voteCount} votes</VoteCount>
+          <VoteCount>{voteCount} </VoteCount>
```

Comment:
Compare yourself above, is the space needed after closing curly brace?

Reply:
I didn't notice it 😆  . I'm already correcting